lk2nd/include/lk2nd/gpio.h

```diff
@@ -9,6 +9,8 @@
 #define LK2ND_GPIO_DEV_TLMM	1
 #define LK2ND_GPIO_DEV_PM8X41	2
 #define LK2ND_GPIO_DEV_PM8X41_PON	3
+#define LK2ND_GPIO_DEV_PM8921	4
+#define LK2ND_GPIO_DEV_PM8921_PON	5
```

Comment:
Hmm, I was kind of expecting this to be `PMIC` and `PMIC_PON`. No platform will ever have both of them and this would likely be more intuitive for people writing the DT part.